tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -73,7 +73,7 @@ ChunkProductionKey = 2508733236
 ChunkProofs = 4130187750
 ChunkState = 1435093277
 ChunkStateTransition = 307448170
-ChunkStateWitness = 94504878
+ChunkStateWitness = 655135716
```

Comment:
I changed the names of some fields in the `ChunkStateWitness`, hence I had to regenerate the schema.

Reply:
Doesn't that mean a change in protocol anyway? On one hand Borsh doesn't encode field names so supposedly that should have no effect (in which case the tool should be adjusted IMO,) on the other hand I'm not familiar with borsh derives enough to be super confident that it really has no effect.

Regardless, rustc has `#[deprecated = "reason"]` that can be put on top of pretty much anything you can think of.

Reply:
renaming field in borsh-serialised struct is not a protocol change

Reply:
Added the deprecated attribute.